### PR TITLE
Improve frontend

### LINKED ISSUE #11: Rubrik, datum och valt SM sparas in vid "cachat" innehåll i motionen
Utifall att man reloadar sidan (eller hoppar tillbaka från att ha skapat en motion) så försvinner ens valda rubrik, angivet datum samt prefixet till "-SM" som skrivits in.

### LINKED ISSUE #12: Öppna PDF i ny flik
Om man trycker på "PDF" så öppnar den PDF-versionen på samma flik som man skrev motionen i, potentiellt vore det najs att få öppna den i en annan flik så man har kvar "arbetsrutan".

### LINKED ISSUE #13: Cleara innehåll
I och med att informationen sparas mellan sid-reloads så vore det trevligt att kunna cleara bort den infon, typ med en knapp eller något.

### PR DESCRIPTION
**Pull request by Herkarl**
- attsatser
  - add numbers to attsatser
  - make attsatser not horizontally resizable
  - make attsatser correctly indented
- better alignment for sm-info
- make it so that all inputs persists when refreshing/reopening the page
- add clear button
- make pdf:s open in new tab

Motioner should look basically the same in the frontend as in the generated pdf:s now

Fixes #13 
Fixes #12
Fixes #11

Tested and works locally

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -28,13 +28,30 @@
...
             </div>
             <div class="col-md-8">
-              <h2>Motioner</h2>
             </div>
             <div class="header-right col-md-2">
-              <a href="#" @click="submitpdf" class="primary-action">PDF</a>
+              <a href="#" @click="submitpdf" class="primary-action">
+                <div class="buttontext">
+                  PDF
+                </div>
```
Comment (niklasvatn) [inline templates/index.html:43]: Does creating PDF work? I had problems with it last week.

Comment (Herkarl) [reply templates/index.html:43]: Yes, it was fixed when deploying #14

Comment (Herkarl) [reply templates/index.html:43]: There are some unintuitive behaviour/bugs where some inputs results in PDF creation failing though, which I intend to look at at some later point

**Review by elmaxe — APPROVED** [review]
LGTM

**Comment by Herkarl** [comment]
Deployed and works

**Pull request merged**